web/src/components/header-btn.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+export default class HeaderBtn extends React.Component {
+  
+  render() {
+      return (
+        <button className={this.props.isOpen ? "header-btn header-btn-open" : "header-btn header-btn-closed"} onClick={this.props.handleClick.bind(this)}>
```

Comment:
as mentioned below, the click can directly be toggled inside this component, and then you can set the `className` of the div below according to that state.